client/components/AutocompleteInputBox/AutocompleteInputBox.tsx

```diff
@@ -1,7 +1,15 @@
 import React, { Component } from 'react'
 import cx from 'classnames'
 
-class AutocompleteInputBox extends Component {
+import { PropsWithClassName } from '../../../types'
+
+type AutocompleteInputBoxProps = React.PropsWithChildren<
+  PropsWithClassName<{
+    footer?: React.ReactNode
+  }>
+>
```

Comment:
You are looking at the outdated code that is no longer part of this PR. Please review the up-to-date version of the code.

Reply:
Ah, so this comment was added much earlier but I didn't click on request changes, which is why it never made it to the PR until an hour back. Ignore this. This part of github's UX does throw me off quite a bit since I'm used to bitbucket.